iree/turbine/kernel/wave/templates/extend_attention.py

```diff
@@ -69,6 +69,17 @@ def get_extend_attention_kernel(
     STORE_ELEMS_PER_THREAD = tkl.sym.STORE_ELEMS_PER_THREAD
 
     SEQ_TILE_SIZE = shape.block_size
+    N_KV_SCALE = 2
+
+    if SEQ_TILE_SIZE is None:
+        # Apply tile heuristic.
+        if shape.max_seq_len <= 128:
```

Comment:
Does our untitests cover both paths?

Reply:
No currently SEQ_TILE_SIZE is always set and we dont do a full sweep. But I think we should add that to our perf tests @nithinsubbiah

Reply:
Do we want to add other attention variants to perf test as well? My understanding was we check perf for only vanilla attention for the interested sizes

Reply:
Let's start with that and once you have that we can add some extend attention tests as well.